src/config.ts

```
@@ -2,27 +2,36 @@ import { z } from "zod";
+/**
+ * Enum for supported output formats
+ */
+export enum OutputFormat {
+  JSON = "json",
+  CSV = "csv",
+  MARKDOWN = "markdown",
+}
+
 configDotenv();
 
 const Page: z.ZodType<Page> = z.any();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Page'. Consider to delete it or rename it.

'Page' is defined here:


(lint/suspicious/noRedeclare)